### DISCUSSION TITLE
[Esame 17 aprile 2024] Esercizio 1

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#458] [Esami M-Z]
![17 apr es.1](https://github.com/user-attachments/assets/950535be-039f-4fc8-b3d8-5284e0696d9c)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/bd973d93-c739-4b80-8038-67310329915d)

**Comment by AglaiaNorza** [comment]
alternativamente

![image](https://github.com/user-attachments/assets/db817f24-fad1-4a64-8bf6-2c1056aaf1a1)
